Code/Framework/AzCore/AzCore/Asset/AssetContainer.cpp

```diff
@@ -32,7 +32,6 @@ namespace AZ::Data
                 "end up in a perpetual loading state if there is no top-level container signalling the completion of the full load.");
         }
 
-        AssetBus::MultiHandler::BusDisconnect();
```

Comment:
should we even keep the AssetBus since it is possible to get it into a dead lock state?

Reply:
I'm not sure I follow.  AssetBus should have been entirely removed from the AssetContainer.  It should only be using AssetLoadBus (the private ebus)